jest.config.js

```diff
@@ -50,6 +50,10 @@ module.exports = {
     'eslint/lib/rules/(.*)': '<rootDir>/node_modules/eslint/lib/rules/$1',
     '^react-native(.*)': '<rootDir>/packages/react-native$1',
   },
+  modulePathIgnorePatterns: [
+    // [macOS] ignore packages that differ from facebook/react-native, since we publish these to verdaccio
+    '<rootDir>/packages/virtualized-lists/',
+  ],
```

Comment:
Just adding a note fro your offline talks, it seems our new approach is to try and not use `verdaccio` for anything except `react-native-macos-init` tests, and this will eventually be removed?

Reply:
Yep, b26a8e9 removes this.